docs/plans/screenshot-pipeline-and-conventions/01-implementer-runbook.md

```
@@ -0,0 +1,203 @@
+## 1c. Commit
+
+```bash
+git add docs/images/dashboard.png README.md tools/simulation/ui/specs-screenshots/screenshots.spec.ts
+git commit -m "docs: commit the dashboard screenshot and show it in the README (#660, #663)"
+```
+
+---
+
+# INCREMENT 2 — capture the palette × theme matrix (#664)
+
+#664 asks for a **visual check** of #651/#652 in night mode and the three non-default farm palettes. It
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)